shell/platform/darwin/macos/BUILD.gn

```diff
@@ -317,3 +321,32 @@ if (build_glfw_shell) {
     public_configs = [ "//flutter:config" ]
   }
 }
+
+zip_bundle("_zip_macos_flutter_framework") {
```

Comment:
If the '_' is meant to communicate something about the visibility of this target, then that would be better accomplished by adding `visibility = [ ":*" ]`.

Reply:
Done! updated the zip_bundle template to include the visibility.